docs/r2/tensorboard_in_notebooks.ipynb

```diff
@@ -110,6 +110,25 @@
         "[1]: https://github.com/ipython/ipykernel/issues/395#issuecomment-479787997\n"
       ]
     },
+    {
+      "metadata": {
+        "id": "9w7Baxc8aCtJ",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In case you are running a [Docker](https://docs.docker.com/install/) image of [Jupyter Notebook server using TensorFlow's nightly](https://www.tensorflow.org/install/docker#examples_using_cpu-only_images), it is necessary to expose not only the notebook's port, but the TensorBoard's port.\n",
+        "\n",
+        "Thus, run the container with the following command:\n",
+        "\n",
+        "```\n",
+        "docker run -it -p 8888:8888 -p 6006 \\\n",
```

Comment:
This doesn’t seem to work for me—when I use `%tensorboard` in the
Jupyter notebook, I still get “localhost refused to connect”. It works
if I change `-p 6006` to `-p 6006:6006`. Can/should we use that instead?

(I’m not super familiar with Docker, so if I’m missing something then
please fill me in!)

Reply:
Sorry, it was really missing ` :6006` in command. I've already fixed.